contracts/src/v0.8/operatorforwarder/test/FuzzOperator.t.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity ^0.8.19;
+
+import "./testhelpers/Deployer.t.sol";
+import "../AuthorizedReceiver.sol";
+import "../Operator.sol";
+
+contract FuzzTokenReceiver is Test {
+  bytes4 private constant ORACLE_REQUEST_SELECTOR = Operator.oracleRequest.selector;
+  bytes4 private constant OPERATOR_REQUEST_SELECTOR = Operator.operatorRequest.selector;
+  uint256 private constant SELECTOR_LENGTH = 4;
+  uint256 private constant EXPECTED_REQUEST_WORDS = 2;
+  uint256 private constant MINIMUM_REQUEST_LENGTH = SELECTOR_LENGTH + (32 * EXPECTED_REQUEST_WORDS);
+  bytes receivedData;
+
+  address _spoofedLinkToken = address(0xCCBBAA);
+
+  function test_FuzzOnTokenTransferFromLink(
```

Comment:
This entire test feels convoluted and I'm not sure what it's supposed to do. It looks like you're testing the test, not the contract code?

Reply:
Yeah I should've read this before adding it in. Removing now, no idea what it does.